modules/server/src/main/scala/scaladex/server/route/FrontPage.scala

```diff
@@ -69,18 +61,146 @@ class FrontPage(env: Env, database: WebDatabase, searchEngine: SearchEngine)(imp
+          case (currentVersion, otherVersions) =>
+            EcosystemHighlight(
+              "Scala.js",
+              Some("Robust Scala-to-JavaScript compiler for browser and node.js."),
+              currentVersion,
+              otherVersions,
+              logo = Option(Url("https://www.scala-js.org/assets/img/scala-js-logo.svg"))
+            )
+        }
+      }
+
+      val (scala2, scala3) = scalaVersions
+        .map {
+          case (scalaVersion, count) =>
+            EcosystemVersion(
+              version = scalaVersion.version,
+              deprecated = scalaVersion.isDeprecated,
+              libraryCount = count,
+              search = Url(s"search?languages=${scalaVersion.label}")
+            )
+        }
+        .partition(_.version < Scala.`3`.version)
+
+      val scala3Ecosystem =
+        selectCurrent(scala3).map {
+          case (current, other) =>
+            EcosystemHighlight(
+              ecosystem = "Scala",
+              description = None,
+              currentVersion = current,
+              otherVersions = other,
+              logo = Some(Url("https://www.scala-lang.org/resources/img/frontpage/scala-spiral.png"))
+            )
+        }
+
+      val scala2Ecosystem =
+        selectCurrent(scala2).map {
+          case (current, other) =>
+            EcosystemHighlight(
+              ecosystem = "Scala",
+              description = None,
+              currentVersion = current,
+              otherVersions = other,
+              logo = Some(Url("https://www.scala-lang.org/resources/img/frontpage/scala-spiral.png"))
+            )
+        }
+
+      val snEcosystem = {
+        val allVersions = scalaNativeVersions.map {
+          case (sn, cnt) =>
+            EcosystemVersion(
+              version = sn.version,
+              deprecated = sn.isDeprecated,
+              libraryCount = cnt,
+              search = Url(s"search?platforms=${sn.label}")
+            )
+        }
+        selectCurrent(allVersions).map {
+          case (currentVersion, otherVersions) =>
+            EcosystemHighlight(
+              ecosystem = "Scala Native",
+              description = Some("Run Scala directly on bare metal using LLVM."),
+              currentVersion = currentVersion,
+              otherVersions = otherVersions,
+              logo = Option(Url("https://scala-native.org/en/stable/_static/logo.png"))
+            )
+        }
+      }
+
+      val sbtPluginEcosystem = {
+        val allVersions = sbtVersions.map {
+          case (sbt, cnt) =>
+            EcosystemVersion(
+              version = sbt.version,
+              deprecated = sbt.isDeprecated,
+              libraryCount = cnt,
+              search = Url(s"search?platforms=${sbt.label}")
+            )
+        }
+        selectCurrent(allVersions).map {
+          case (currentVersion, otherVersions) =>
+            EcosystemHighlight(
+              ecosystem = "SBT",
```

Comment:
```suggestion
              ecosystem = "sbt",
```
"sbt" is a proper noun that is always spelled all-lowercase. Even at the start of sentences.